modules/oadp-storage-class-mapping.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/application_backup_and_restore/advanced-topics.adoc
+
+[id="oadp-storage-class-mapping_{context}"]
+:_mod-docs-content-type: CONCEPT
+= Storage class mapping
+
+Storage class mapping allows you to define rules or policies specifying which storage class should be applied to different types of data. This feature automates the process of determining storage classes based on access frequency, data importance, and cost considerations. It optimizes storage efficiency and cost-effectiveness by ensuring that data is stored in the most suitable storage class for its characteristics and usage patterns.
+
+You can use the `change-storage-class-config` field to change the storage class of your data objects. This feature allows you to optimize costs and performance by moving data between different storage tiers, such as from standard to archival storage, based on your needs and access patterns.
+
+[id=storage-class-mapping-mtc_{context}]
+== Storage class mapping with MTC
+
+You can use Migration Toolkit for Containers (MTC) to migrate containers, including application data, from one OpenShift Container Platform cluster to another cluster and for storage class mapping and conversion. You can convert the storage class of a persistent volume (PV) by migrating it within the same cluster. To do so, you must create and run a migration plan in the Migration Toolkit for Containers (MTC) web console.
+
+For more information see link:https://docs.openshift.com/container-platform/4.15/migration_toolkit_for_containers/advanced-migration-options-mtc.html#migration-converting-storage-classes_advanced-migration-options-mtc[Migration converting storage classes]
```

Comment:
```suggestion
For more information see xref:../../backup_and_restore/advanced-migration-options-mtc.adoc#migration-converting-storage-classes_advanced-migration-options-mtc[Migration converting storage classes].
```

Reply:
@anarnold97 
I tried replacing the link with the xref but it didn't work and it broke the build.

Reply:
Well, fix the xref